forge-gui/res/cardsfolder/s/sontaran_general.txt

```diff
@@ -5,9 +5,8 @@ PT:5/5
 K:Trample
 K:Haste
 T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | IsPresent$ Creature.attacking+Other | NoResolvingCheck$ True | PresentCompare$ GE2 | Execute$ TrigGoad | TriggerDescription$ Battalion — Whenever CARDNAME and at least two other creatures attack, for each opponent, goad up to one target creature that player controls. Those creatures can't block this turn. (Until your next turn, they attack each combat if able and attack a player other than you if able.)
-SVar:TrigGoad:DB$ Goad |ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select up to one target creature that opponent controls | TargetMin$ 0 | TargetMax$ OneEach | TargetsWithDifferentControllers$ True | SubAbility$ DBNoBlock
-SVar:DBNoBlock:DB$ Effect | RememberObjects$ Targeted | ForgetOnMoved$ Battlefield | StaticAbilities$ Unblockable
-SVar:Unblockable:Mode$ CantBlockBy | ValidAttacker$ Card.IsRemembered | Description$ Those creatures can't be blocked this turn.
+SVar:TrigGoad:DB$ Goad |ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select up to one target creature that opponent controls | TargetMin$ 0 | TargetMax$ OneEach | TargetsWithDifferentControllers$ True | SubAbility$ DBCantBlock
+SVar:DBCantBlock:DB$ Pump | Defined$ Targeted | KW$ HIDDEN CARDNAME can't block. | IsCurse$ True | SpellDescription$ Target creature can't block this turn.
```

Comment:
while this would work we're slowly moving away from hidden KW
So just keep using the effect and switch the wrong param to `ValidBlocker$` instead

Reply:
can you give me an example where "Creature can't block this turn" is made without the hidden KW?

Reply:
Ahn-Crop Crasher